onnx/defs/generator/defs.cc

```diff
@@ -26,6 +27,36 @@ ONNX_OPERATOR_SET_SCHEMA(
             "The value for the elements of the output tensor in sparse format.",
             AttributeProto::SPARSE_TENSOR,
             false)
+        .Attr(
+            "value_int",
+            "The value for the elements of the output tensor.",
```

Comment:
I would suggest something like:
> The value for the sole element for the scalar, int64, output tensor.

> The values for the elements for the 1D, int64, output tensor.

> The value for the sole element for the scalar, float32, output tensor.

and so on.

This would make it clear what the output tensor will actually look like (shape, element type).